lib/temporal/client/grpc_client.rb

```diff
@@ -206,7 +204,7 @@ def respond_activity_task_completed_by_id(namespace:, activity_id:, workflow_id:
           workflow_id: workflow_id,
           run_id: run_id,
           activity_id: activity_id,
-          result: Serializer::Payload.new(result).to_proto
+          result: Temporal.configuration.converter.to_payloads(result)
```

Comment:
Is this intended not to wrap the result in array like the `respond_activity_task_completed` call?

Reply:
Nope, that's a bug.